WhaleSpotting.UnitTests/Controllers/SightingControllerTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FakeItEasy;
+using FluentAssertions;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WhaleSpotting.Controllers;
+using WhaleSpotting.Models.ApiModels;
+using WhaleSpotting.Services;
+using Xunit;
+
+namespace WhaleSpotting.UnitTests.Controllers
+{
+    public class SightingsControllerTests
+    {
+        private readonly SightingsController _underTest;
+        private readonly ISightingService _sightings = A.Fake<ISightingService>();
```

Comment:
Could you move this above the _underTest declaration

Reply:
fixed